### ISSUE TITLE
Dedicated tap mode (or increased touch area) on watch

### ISSUE THREAD

**Issue by liamdawson** [patzly/tack-android#42]
My primary use case for Tack is for swing dancing - I like to tap out a beat to check the BPM of a song. Helps a lot when a band is playing, too.

I found that it's hard to tell whether I might be missing the touch area of the beat button, so it'd be great to have:

- a dedicated timing mode,
- a bigger touch area for the button, and/or
- more obvious visual feedback on tap

Thanks for building Tack!

**Comment by patzly** [comment]
Hi and many thanks! Did you already see the tempo screen which you get when you tap on the tempo? There the tempo tap button is large. Do you talk about this button or about the small version on the main screen?

**Comment by patzly** [comment]
@liamdawson Does it fit your needs? :)

**Comment by liamdawson** [comment]
> @liamdawson Does it fit your needs? :)

Yup, it's not perfectly suited to my use-case (like the dedicated app I used to use on my last Fitbit), but it does the job.

I don't imagine adding a dedicated BPM "checker" feature would suit most users, so I'm happy to call this closed and use it as-is.

**Comment by patzly** [comment]
Let me see if I can implement an extra fullscreen mode which opens when the user keeps the tempo tap button pressed :)

**Comment by patzly** [comment]
@liamdawson Will be available with v4.8.0 on your watch!

**Comment by patzly** [comment]
Note for me: implemented in 16a2b7b285f24776b968c4f75631527eda937d97

**Comment by patzly** [comment]
@liamdawson The Wear OS update is finally out!
